app/assets/javascripts/components/story/StoryAttachment.js

```diff
@@ -34,6 +34,14 @@ class StoryAttachment extends React.Component {
     })(this, this.progressElementId, this.finishedElementId));
   }
 
+  isEmpty(obj) {
```

Comment:
@talyssonoc Does not _underscore_ have a function for that?

Reply:
Yep it does! It's here http://underscorejs.org/#isEmpty

Reply:
Thanks, Its done